x/group/keeper/msg_server.go

```diff
@@ -337,6 +336,7 @@ func (k Keeper) CreateGroupPolicy(goCtx context.Context, req *group.MsgCreateGro
 
 		if k.accKeeper.GetAccount(ctx, accountAddr) != nil {
 			// handle a rare collision
+			// TODO should we throw an error here, because there's duplicate grup policy address
```

Comment:
Not sure about here. Should we instead throw an error?

Reply:
But don't we handle this with `continue` which will loop over again and try to derive a new `accountAddr`

Reply:
True. I added a comment in the code to explain this.